pkg/gen/input/workflows/internal/file/cluster_app_schema_validation.yaml.template

```diff
@@ -1,7 +1,9 @@
 {{{{ .Header }}}}
 
-# This workflow che
```

Comment:
![che](https://user-images.githubusercontent.com/56017655/229730524-f66a1cdd-fcd2-47e7-9c40-271947bd59f0.jpg)

Reply:
This killed me